src/kube-runtime/src/plugins/cmd/init.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/local/bin/python
```

Comment:
```suggestion
#!/usr/bin/python
```

Reply:
The base image python-alpine we use, python is under /user/local/bin/python